doc/docs/master/reference/scaling_changes.md

```diff
@@ -0,0 +1,70 @@
+# Pipeline Scaling Limitation in Community Edition
+
+!!! Warning "We are here to help!" 
+    Scaling Pachyderm pipelines is easy to do but hard for users to perfect. 
+
+It can be difficult to balance the breaking down of data science systems into multiple pipelines, horizontally scaling those pipelines to optimize the processing time and effectively understanding Pachyderm's unique data-first approach to development in connecting those pipelines. Lalalal
+
+Starting with this new 1.13.0 release, we introduce a **limit to the number of pipelines and their parallelization** Community Edition users can deploy.
```

Comment:
This phrasing seems clumsy. I wouldn't say it starts with this release, as this is the documentation for the current release and is not included in the others.

Reply:
Copy/Paste from the PRD - Removed -